manim/mobject/mobject.py

```diff
@@ -1140,6 +1427,34 @@ def put_start_and_end_on(self, start, end):
 
     # Background rectangle
     def add_background_rectangle(self, color=BLACK, opacity=0.75, **kwargs):
```

Comment:
import `Colors` first from the root and then
```suggestion
    def add_background_rectangle(self, color: Colors=BLACK, opacity: float=0.75, **kwargs):
```